data_steward/ci/setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Create buckets
+gsutil mb -c nearline -p ${APPLICATION_ID} gs://${DRC_BUCKET_NAME}
+gsutil mb -c nearline -p ${APPLICATION_ID} gs://${BUCKET_NAME_FAKE}
```

Comment:
Should we make all our testing happen in buckets that are nearline?

Reply:
Didn't think about that. Now I'm thinking of changing it to `us-east4` (see [Bucket Locations](https://cloud.google.com/storage/docs/bucket-locations)). What do you think?